api-java/samples/server/pom.xml

```diff
@@ -77,5 +77,55 @@
 
     <build>
         <sourceDirectory>${project.basedir}/trinsic</sourceDirectory>
+        <plugins>
```

Comment:
Why not just use .sh if you're already executing a script there, similar as other languages?